plugins/network-policy-audit/scripts/netpol_analyzer_cli.py

```
@@ -0,0 +1,228 @@
+        parser_obj = NetworkPolicyParser(
+            namespace=args.namespace if not cluster_wide else None,
+            cluster_wide=cluster_wide,
+            kubeconfig=args.kubeconfig
+        )
+
+        # Fetch policies
+        logger.info("Fetching NetworkPolicies from cluster...")
+        policies_raw = parser_obj.get_policies()
+
+        if not policies_raw:
+            print(f"\n⚠️  No NetworkPolicies found", file=sys.stderr)
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)